src/pattern-library/routes.js

```diff
@@ -193,9 +196,19 @@ const routes = [
   { title: 'Table', group: 'data' },
   { title: 'Dialog', group: 'feedback' },
   { title: 'Spinner', group: 'feedback' },
-  { title: 'Button', group: 'input' },
+  {
+    title: 'Button',
+    group: 'input',
+    component: ButtonsPage,
+    route: '/input-button',
+  },
   { title: 'Checkbox', group: 'input' },
-  { title: 'IconButton', group: 'input' },
+  {
+    title: 'IconButton',
+    group: 'input',
+    component: IconButtonPage,
+    route: '/input-iconbutton',
```

Comment:
I noticed the URL for these links renders incorrectly on hover, though it does navigate to the correct URL:

<img width="342" alt="Wrong hover URL" src="https://user-images.githubusercontent.com/2458/180795887-365666c8-eea6-4e99-9a9d-2df3e40d8dd5.png">

Reply:
This is coming from the `href` attribute:

<img width="610" alt="Wrong link href" src="https://user-images.githubusercontent.com/2458/180796113-be59fb40-81cb-4e55-9ba3-abec3f0104ec.png">

Reply:
Well-spotted small typo in a `Library` component; easily fixed.